events/events.go

```diff
@@ -19,12 +19,6 @@ type Forwarder interface {
 	Forward(ctx context.Context, envelope *events.Envelope) error
 }
 
-type publisherFunc func(ctx context.Context, topic string, event Event) error
```

Comment:
I think this is used in tests.

Reply:
Are there tests that don't run as part of CI? CI seems to be green.